### DISCUSSION TITLE
Change mouse cursor to hand icon when pointed over a markup (HTML) button

### DISCUSSION THREAD

**Discussion by dulajra** [h2oai/wave#956] [Ideas] [answered]
With markup support and newly added JS support, we can implement action elements (ex: icon buttons) with Wave. But users won't feel those action elements are clickable because the mouse cursor is not changed when those elements are focused. 

Is there any way to let the user feel those elements are interactive?

**Comment by mturoci** [comment, marked as answer]
Can you provide a minimal code? Since you have a full JS support, you can add the styles that indicate clickability yourself, e.g. `element.style.cursor = 'pointer'`

**Reply by dulajra** [reply]
@mturoci Thanks for the hint. I was able to achieve the same by wrapping the element with an `<a></a>`

**Reply by lo5** [reply]
@dulajra might be better to author components in Wave that do what you want instead of resorting to HTML/JS in your Python app, which is harder to maintain.

**Reply by dulajra** [reply]
@lo5 Yes I also feel that it will be harder to maintain if we have more HTML and JS in long term apps like MLOps.
